mycore-webtools/src/main/resources/META-INF/resources/modules/webtools/session/js/sessionListing.js

```diff
@@ -142,12 +142,25 @@ mycore.session.listing = {
     var session = mycore.session.listing.getSession(id);
     var stacktrace = session.constructingStacktrace.stacktrace;
     var header = dialog.parents(".modal-dialog").find(".modal-header");
+    var html = "ID: " + session.id;
```

Comment:
The whole function is very dangerous and allows to inject javascript code in to the session listing. Example:
```
curl -H "User-Agent: <script>alert('test')</script>" http://localhost:8291/mir/content/index.xml
```
Open the http://localhost:8291/mir/modules/webtools/session/sessionListing.xml as admin and click the colored item of the right session and you get a funny message box. 


To avoid this you need to escape the content. 
```
    let cleanerElement = document.createElement('div');
    let badText = '<script>alert("XSS");</script>'
    cleanerElement.textContent = badText;
    let cleanText = badText.innerHTML;
```

Reply:
You're absolutely right. Will fix.